tests/mpd/test_translator.py

```diff
@@ -79,6 +80,31 @@ def test_track_to_mpd_format_for_nonempty_track(self):
         self.assertNotIn(('Comment', 'a comment'), result)
         self.assertEqual(len(result), 14)
 
+    def test_track_to_mpd_format_with_last_modified(self):
+        track = self.track.replace(last_modified=995303899000)
+        # Due to this being local time-zone dependant, we have
+        # to calculate what the Last-Modified output will look like
+        # on the machine where the tests are being run. So this only tests
+        # that the output is actually there.
+        datestring = datetime.datetime.fromtimestamp(
+            track.last_modified // 1000).isoformat()
+        result = translator.track_to_mpd_format(track)
+        self.assertIn(('file', 'a uri'), result)
+        self.assertIn(('Time', 137), result)
+        self.assertIn(('Artist', 'an artist'), result)
+        self.assertIn(('Title', 'a name'), result)
+        self.assertIn(('Album', 'an album'), result)
+        self.assertIn(('AlbumArtist', 'an other artist'), result)
+        self.assertIn(('Composer', 'a composer'), result)
+        self.assertIn(('Performer', 'a performer'), result)
+        self.assertIn(('Genre', 'a genre'), result)
+        self.assertIn(('Track', '7/13'), result)
+        self.assertIn(('Date', '1977-01-01'), result)
+        self.assertIn(('Disc', 1), result)
+        self.assertIn(('Last-Modified', datestring), result)
```

Comment:
I think datestring should be written out here, instead of calculating it with the same logic as used in the implementation.